src/components/user-list/index.js

```diff
@@ -26,6 +26,7 @@ const UserList = () => {
             <ListItemIcon>
               <VotingIcon hasVoted={user.hasVoted} />
             </ListItemIcon>
+            <ListItemText primary={user.vote} />
```

Comment:
Do you show it now, or in a table at the end to not influence anyone else?

Reply:
good call, will adjust

Reply:
Your lack of concern for the integrity of the voting process is concerning @jrosbrow